tests/filecheck/mlir-conversion/with-mlir/dialects/memref/memref_ops_custom.mlir

```diff
@@ -46,4 +48,7 @@ memref.dealloc %a1 : memref<2x3xf32>
 // CHECK-NEXT:   %{{.*}} = memref.alloc() : memref<2x3xf32>
 // CHECK-NEXT:   %{{.*}} = memref.alloc()[%{{.*}}] {alignment = 8 : i64} : memref<2x3xf32, affine_map<(d0, d1)[s0] -> (d0 + s0, d1)>, 1>
 // CHECK-NEXT:   memref.dealloc %{{.*}} : memref<2x3xf32>
+// CHECK-NEXT:   %{{.*}} = memref.subview %{{.*}}[0, 0] [1, 3] [1, 1] : memref<10x3xi32> to memref<3xi32>
+// CHECK-NEXT:   %{{.*}} = memref.subview %{{.*}}[%{{.*}}, 0] [1, 3] [1, 1] : memref<10x3xi32> to memref<3xi32, strided<[1], offset: ?>>
```

Comment:
@PapyChacal, this is the output from MLIR. It looks like the doc online disagrees, but I'd rather go with the MLIR we have as source of truth.